packages/jsii-diff/lib/stability.ts

```diff
@@ -0,0 +1,45 @@
+import reflect = require('jsii-reflect');
+import spec = require('jsii-spec');
+import { ApiElement, ComparisonContext } from './types';
+
+export function compareStabilities(original: reflect.Documentable & ApiElement, updated: reflect.Documentable, context: ComparisonContext) {
+  // Nothing to do in these cases
+  if (original.docs.stability === undefined || original.docs.stability === updated.docs.stability) { return; }
+
+  // Not allowed to disavow stability
```

Comment:
Is that the case where stability is inherited? What happens if you change the stability of the oRent module/type, is it inherited by the Jsii compiler! Can we evened even have an undefined stability for an api?

Also, wasn’t this what we wanted to do for the CFN layer?

Reply:
Inheritance has already been calculated at this level. I thought for the CFN layer we introduce a new stability level.